pkg/pulsar/utils/data.go

```diff
@@ -183,7 +183,7 @@ type NamespacesData struct {
 }
 
 type TopicStats struct {
-	BacklogSize         int64                        `json:"backlogSize"`
+	BacklogSize         uint64                       `json:"backlogSize"`
```

Comment:
The Pulsar defines the `backlogSize` as a `long` type, so I don't suggest you change this type.